examples/prometheus/rules/sync-gateway.rules.yml

```diff
@@ -4,37 +4,86 @@ groups:
       - record: sgw::gsi::total_queries
         expr: sum by (instance, database, job) ({__name__=~"sgw_gsi_views_.*_count"})
       - alert: TooManyAuthFailuresInLastHour
-        expr: deriv(sgw_security_auth_failed_count[1h]) > 1000
+        expr: increase(sgw_security_auth_failed_count[1h]) > 1000
         for: 1m
         labels:
           severity: warning
         annotations:
           summary: Too Many Auth Failures in Last Hour
       - alert: TooManyDocumentAccessFailuresInLastHour
-        expr: deriv(sgw_security_num_access_errors[1h]) > 1000
+        expr: increase(sgw_security_num_access_errors[1h]) > 1000
         for: 1m
         labels:
           severity: warning
         annotations:
           summary: Too many Document Access Failures in Last Hour
       - alert: TooManyDocumentRejectionFailuresInLastHour
-        expr: deriv(sgw_security_num_docs_rejected[1h]) > 1000
+        expr: increase(sgw_security_num_docs_rejected[1h]) > 1000
         for: 1m
         labels:
           severity: warning
         annotations:
           summary: Too many Document Rejection Failures in Last Hour
       - alert: HighRevCacheMissRate
-        expr: sgw_cache_rev_cache_misses / (sgw_cache_rev_cache_misses + sgw_cache_rev_cache_hits) >= 1.0
+        expr: sgw_cache_rev_cache_misses / (sgw_cache_rev_cache_misses + sgw_cache_rev_cache_hits) >= 0.8
         for: 1m
         labels:
           severity: warning
         annotations:
           summary: High Rev Cache Miss Rate
       - alert: HighChannelCacheMissRate
-        expr: sgw_cache_chan_cache_misses / (sgw_cache_chan_cache_misses + sgw_cache_chan_cache_hits) >= 1.0
+        expr: sgw_cache_chan_cache_misses / (sgw_cache_chan_cache_misses + sgw_cache_chan_cache_hits) >= 0.8
         for: 1m
         labels:
           severity: warning
         annotations:
           summary: High Channel Cache Miss Rate
+      - alert: HighDeltaCacheMissRate
+        expr: sgw_delta_sync_delta_sync_miss / (sgw_delta_sync_delta_sync_miss + sgw_delta_sync_delta_cache_hit) >= 0.8
```

Comment:
As discussed, confirm behaviour when stats aren't present (e.g. div by zero)

Reply:
This is handled
- Absent stats are checked for and handled
- Divide by zero returns NaN as expected and doesn't trigger the alert